bin/localeapp

```diff
@@ -128,9 +128,15 @@ module LocaleappGLIWrapper
   end
 
   desc "Pulls all translations from localeapp.com"
+  arg_name "[target_dir]"
```

Comment:
The current convention in this file appears to be that `<>` is used for _required_ arguments, not optional ones.